constants.py

```diff
@@ -10,3 +10,9 @@
 
 S_TO_MS = 1000.0
 MS_TO_S = 0.001
+
+DATA_PACKET_SIZE = 1024.0 * BYTE_TO_BIT
+ACK_PACKET_SIZE = 64.0 * BYTE_TO_BIT
+ROUT_PACKET_SIZE = 128.0 * BYTE_TO_BIT
+
+ACK_DELAY = 500.0 * MS_TO_S
```

Comment:
maybe call this ACK_TIME_OUT? that might be clearer cuz i couldn't tell what this meant until i read the later code